dvc/remote/azure.py

```diff
@@ -16,23 +15,36 @@
 class AzureRemoteTree(BaseRemoteTree):
     scheme = Schemes.AZURE
     PATH_CLS = CloudURLInfo
-    REQUIRES = {"azure-storage-blob": "azure.storage.blob"}
+    REQUIRES = {
+        "azure-storage-blob": "azure.storage.blob",
+        "azure-cli-core": "azure.cli.core",
+    }
     PARAM_CHECKSUM = "etag"
     COPY_POLL_SECONDS = 5
     LIST_OBJECT_PAGE_SIZE = 5000
 
     def __init__(self, repo, config):
+        # NOTE: az_config takes care of env vars
+        from azure.cli.core.extension import az_config
```

Comment:
This one is probably not meant for external use, so again probably better to use knack directly.